Content.Shared/Atmos/Components/GasPressurePumpComponent.cs

```diff
@@ -0,0 +1,25 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Atmos.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class GasPressurePumpComponent : Component
+{
+    [DataField, AutoNetworkedField]
+    public bool Enabled = true;
+
+    [DataField("inlet")]
+    public string InletName = "inlet";
+
+    [DataField("outlet")]
+    public string OutletName = "outlet";
+
+    [DataField, AutoNetworkedField]
+    public float TargetPressure = Atmospherics.OneAtmosphere;
```

Comment:
Does this leak the target pressure setpoint over the network when the gas pump is out of examine range but still inside PVS? This doesn't seem to me like an important issue, but we should probably be aware of what variables we are sending to the client. Perhaps we don't need the out of range examine check anymore?